interpreter/cling/lib/Interpreter/DynamicLibraryManagerSymbol.cpp

```diff
@@ -541,6 +542,27 @@ namespace cling {
       }
     }
 
+    // On Windows, the symbols are present in COFF format.
+    if (BinObjFile->isCOFF()) {
+      llvm::object::COFFObjectFile* CoffObj = cast<llvm::object::COFFObjectFile>(BinObjFile);
+
+      // In COFF, the symbols are not present in the SymbolTable section
+      // of the Object file. They are present in the ExportDirectory section.
+      export_directory_iterator I = CoffObj->export_directory_begin();
+      export_directory_iterator E = CoffObj->export_directory_end();
+
+      for (; I != E; I = ++I) {
```

Comment:
That should become:

`for (auto I = CoffObj->export_directory_begin(), E = CoffObj->export_directory_end();
         I != E; I = ++I) {`